regression/cbmc/dereference-cache-flag/README.md

```diff
@@ -0,0 +1 @@
+This is just checking that the --symex-cache-dereferences flag actually turns dereference caching on
```

Comment:
I'm not a huge fan of README files in tests and would rather have all information in the .desc files, but that's maybe just me. More importantly, I do wonder how this test is testing that when it's really the overarching Makefile that's enabling this. Shouldn't the .desc file explicitly include that flag (even when that would be redundant)?

Reply:
Fixed in incoming commit.